website/docusaurus.config.js

```diff
@@ -112,7 +112,7 @@ const config = {
     description:
       'An optimized site generator in React. Docusaurus helps you to move fast and write content. Build documentation websites, blogs, marketing pages, and more.',
   },
-  staticDirectories: ['static', '_dogfooding/_asset-tests'],
+  staticDirectories: ['static', path.join(__filename, '_dogfooding/_asset-tests')],
```

Comment:
probably __dirname instead of __filename?

Or just `path.resolve('_dogfooding/_asset-tests')` ?

Reply:
Oops, my bad😅 Was in a rush